docs/articles/basics.md

```diff
@@ -83,12 +83,14 @@ Similarly, the Docker client can use `-H` to connect to a custom port.
 
 `-H` accepts host and port assignment in the following format:
 
-    tcp://[host][:port] or unix://path
+    tcp://[host][:port][path] or unix://path
```

Comment:
@LK4D4 to me the / is included in the path. I can use ` tcp://[host][:port][/path]` if you'd like